catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/QueryApplication.java

```diff
@@ -109,6 +137,34 @@ public void init() {
           res.header("Content-Encoding", "gzip");
         });
 
+    post(
+        "/cql/transform/:transformerId",
+        (req, res) -> {
+          String transformerId = req.params(":transformerId");
+          String body = util.safeGetBody(req);
+          CqlRequest cqlRequest = mapper.readValue(body, CqlRequest.class);
+          CqlQueryResponse cqlQueryResponse = executeCqlQuery(cqlRequest);
+
+          LOGGER.trace("Finding transformer to transform query response.");
+
+          ServiceReference<QueryResponseTransformer> queryResponseTransformer =
```

Comment:
i get an unchecked assignment from ServiceReference to ServiceReference<QueryResponseTransformer>, unsure if this is going to be an issue if left unchecked.

Reply:
I don't think it will be possible for OSGi to return the wrong interface; no an unchecked assignment should be fine in this case

Reply:
You could change `queryResponseTransformers` to be a `List<ServiceReference<QueryResponseTransformer>>`.